getting-started/vs-code-integration/introduction.md

```diff
@@ -8,28 +8,28 @@ position: 1
 
 # Visual Studio Code Integration Overview
 
-The **Telerik UI for Blazor Template Wizard** [Visual Studio Code](https://code.visualstudio.com/) Extension enhances the experience in developing Blazor web applications with Telerik UI for Blazor.
+The **Telerik UI for Blazor Template Wizard** Extension for [Visual Studio Code](https://code.visualstudio.com/) enhances the experience in developing Blazor web applications with Telerik UI for Blazor.
 
+{% if site.has_cta_panels == true %}
+{% include cta-panel-introduction.html %}
+{% endif %}
 
->caption The Telerik Blazor Extension for Visual Studio Code provides the following advantages:
```

Comment:
If you need another heading level, use the corresponding heading instead of a caption.